draft-rescorla-tls-esni.md

```diff
@@ -635,17 +678,21 @@ to this document.
 --- back
 
 
-# Communicating SNI to Backend Server {#communicating-sni}
+# Communicating SNI and Nonce to Backend Server {#communicating-sni}
 
 As noted in {{backend-server-behavior}}, the backend server will
-generally not know the true SNI in Split Mode. It is possible for
-the client-facing server to communicate the true SNI to the backend server,
-but at the cost of having that communication not be unmodified TLS 1.3.
-The basic idea is to have a shared key between the client-facing server
-and the backend server (this can be a symmetric key) and use it to
-AEAD-encrypt Z and send the encrypted blob at the beginning of the connection before
+generally not know the true SNI in Split Mode, nor will it know 
+PaddedServerNameList.nonce. The former is not needed for the handshake
+to complete with a default certificate, while the latter is mandatory
+for successful handshake completion. Thus, client-facing servers must
+be able to communicate at least the nonce to backend servers.
```

Comment:
This text also seems OBE.

Reply:
Yep, addressed, and also removed the hand-wavy bit.